cmd/cloud/server.go

```diff
@@ -90,9 +91,10 @@ var serverCmd = &cobra.Command{
 		}
 
 		clusterSupervisor, _ := command.Flags().GetBool("cluster-supervisor")
-		clusterInstallationSupervisor, _ := command.Flags().GetBool("cluster-installation-supervisor")
+		groupSupervisor, _ := command.Flags().GetBool("group-supervisor")
 		installationSupervisor, _ := command.Flags().GetBool("installation-supervisor")
-		if !clusterSupervisor && !installationSupervisor && !clusterInstallationSupervisor {
+		clusterInstallationSupervisor, _ := command.Flags().GetBool("cluster-installation-supervisor")
+		if !clusterSupervisor && !installationSupervisor && !clusterInstallationSupervisor && !groupSupervisor {
```

Comment:
Could we break this check into 3 and log more-specifically what won't work if each supervisor is disabled? 0/5 on this, just a thought.

Reply:
Do you mean 4 or more checks? One for each supervisor?

The logic check here is less about warning that a supervisor is disabled and more about warning that all supervisors are disabled. If that were true the server would just be doing API stuff like I mentioned in the log.

Do you think this should be changed so individual warnings are given? I don't want to clog up the server startup logs with warnings that sorta aren't warnings but only 0/5.

Reply:
>Do you think this should be changed so individual warnings are given?
I'm actually not sure, I guess my concern was that the user might not be aware of limitations imposed if one of the supervisors (but not all) was disabled.

In case my original comment was unclear, I wasn't asking you to break it into three checks, I was looking to bring your attention to the log message and ask, really ask, if you thought we _should_ do that. If it doesn't make sense to you to do that, just resolve this conversation and move on :)